aries-site/src/data/wcag/components.json

```
@@ -862,5 +862,519 @@
         "status": "not-applicable"
       }
     ]
+  },
+  {
+    "dateinput": [ 
+      {
+        "rule": "1.1.1",
+        "label": "Non-text content",
+        "status": "failed",
```

```suggestion
        "status": "conditional",
```
I believe this should be conditional, [see comment here](https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=cqH7WO&nav=MTBfe2ZmYmY3NGE4LTNlZjQtNDBlNS05YjEyLTQ5NWFjY2FmY2E5ZX1fezAzRTJCOTg2LTUzNzAtNEZCMy1CRkZGLTM5OUNCMjVBMTMwNX0)